src/views/blocks/tickets/submit.php

```diff
@@ -24,7 +24,7 @@
 try {
 	/** @var \Tribe__Tickets__Attendee_Registration__Main $attendee_registration */
 	$attendee_registration = tribe( 'tickets.attendee_registration' );
-} catch ( RuntimeException $exception ) {
+} catch ( \Exception $exception ) {
```

Comment:
So just make sure we catch _any_ exception? ✅

Reply:
None of the expections from DI are `RuntimeException` specifically.